lib/seattleflu/id3c/cli/command/clinical.py

```diff
@@ -456,36 +464,198 @@ def add_kp_manifest_data(df: pd.DataFrame, manifest_filenames: tuple, manifest_f
 
 
 @clinical.command("parse-phskc")
-@click.argument("phskc_filename", metavar = "<PHSKC Clinical Data filename>")
-@click.argument("phskc_specimen_manifest_filename", metavar = "<PHSKC Specimen Manifest filename(s)>")
+@click.argument("phskc_manifest_filename", metavar = "<PHSKC Clinical Manifest Data filename>",
+            type = click.Path(dir_okay=False))
+@click.argument("file_pattern", metavar = "<PHSKC Clinical Data filename pattern>")
 @click.argument("geocoding_cache_file", envvar = "GEOCODING_CACHE", metavar = "<Geocoding cache filename>",
             type = click.Path(dir_okay=False, writable=True))
 
-def parse_phskc(phskc_filename: str, phskc_specimen_manifest_filename: str, geocoding_cache_file: str = None) -> None:
+def parse_phskc(phskc_manifest_filename: str, file_pattern: str, geocoding_cache_file: str = None) -> None:
     """
     Process clinical data from PHSKC.
 
-    Given a <PHSKC Clinical Data filename> of an Excel document, a
-    <PHSKC Specimen Manifest Filename> of a newline-delimited JSON document,
-    and a <Geocoding Cache filename> selects specific columns of interest and
-    reformats the queried data into a stream of JSON documents suitable for the
-    "upload" sibling command.
+    Given a path to PHSKC clinical files that need to be parsed, a
+    <PHSKC Clinical Manifest Data filename> and a <Geocoding Cache filename>
+    selects specific columns of interest and parses them into a clinical
+    manifest data file. If data had been previously parsed, compare the
+    last modified timestamps of the file with the last parsed timestamps to decide
+    if we need to parse this file again.
+
+    Clinical records are parsed and transformed into suitable data for downstream
+    CID matching. PII is not removed in this function. Parsed data will later be joined
+    with barcode data present in the LIMS.
+
+    All clinical records (both newly and previously parsed data) are output to stdout
+    as newline-delimited JSON records. You will likely want to redirect stdout to a file.
+    """
+    parsed_clinical_records = pd.read_json(phskc_manifest_filename, orient='records', dtype={'inferred_symptomatic': 'str'}, lines=True)
```

Comment:
Could add a check to make sure `phskc_manifest_filename` file exists. If it doesn't exist, there's an uncaught exception.